docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: "2"
 services:
 
   content:
-    build: .
+    # to build this image locally, execute 'make'
     image: quay.io/giantswarm/docs:latest
```

Comment:
This change is unrelated. I figured that it is misleading to have the build key here, as the `make` command is required.